app.rb

```diff
@@ -0,0 +1,145 @@
+require_relative 'item'
+require_relative 'music_album'
+require_relative 'genre'
+require_relative 'book'
+require_relative 'label'
+require_relative 'storage/author_method'
+require_relative 'storage/games_method'
+require_relative 'storage/book_method'
+require_relative 'storage/genre_method'
+require_relative 'storage/label_method'
+require_relative 'storage/music_album_method'
+
+class App
+  include AuthorsDataController
+  include GamesDataController
+  include BooksDataController
+  include MusicAlbumDataController
+  include GenreDataController
+  include LabelsDataController
+
+  def initialize
+    @music_albums = load_music_albums
+    @games = load_games
+    @genres = load_genres
+    @books = load_books
+    @authors = load_authors
+    @labels = load_labels
```

Comment:
[REQUIRED] When I delete the JSON folder holding the saved data and I try to load the app from scratch I get an error. Kindly ensure that your app starts properly even when the JSON files are not present:
<img width="1440" alt="image" src="https://github.com/Xghayor/comt-console-app/assets/62938735/63f70d9f-bdd3-45c9-9f12-ff3a2bfaec0a">

Reply:
@fobadara Hi,

Thanks for the review, The reason is because you delete the directory which was keeping all the route for json files in code, if you remove the files within the json_file folder, the app won't crash.

Thanks